server/game/cards/09.3-JfS/StrideTheWaves.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class StrideTheWaves extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Move attached character in or out of the conflict',
+            limit: AbilityDsl.limit.perRound(2),
+            condition: context => this.game.isDuringConflict() &&
+                this.game.rings.water.isConsideredClaimed(context.player),
```

Comment:
`this.game` --> `context.game`

Probably doesn't matter, but I think this is preferred.

Reply:
I cleaned it up anyway, seemed better.